### PR TITLE
Omit prefix

### PR DESCRIPTION
**Pull request by jsoref**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +78,7 @@ the following map to the given option
 - `VINNIE_GITLAB_TOKEN` sets `--gitlab-token`
 - `VINNIE_REPO_PUSH` sets `--push`
 - `VINNIE_PREFIX` sets `--prefix`
+- `VINNIE_OMIT_PREFIX` sets `--omit-prefix`
```
Comment (fcurella) [inline README.md:85]: We'll need to also document what this options does.

**Review by fcurella — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jsoref) [reply README.md:85]: https://github.com/revsys/vinnie/pull/9/files#diff-04c6e90faac2675aa89e2176d2eec7d8R62

**Comment by jsoref** [comment]
@fcurella: updated

File: vinnie/base.py [hunk 2]
```diff
@@ -84,7 +88,7 @@ def push(self, remote):
         if self.config.push:
             self.backend.push(remote)
         else:
-            warnings.warn("Skipping push.", UserWarning)
+            print("Skipping push.", file=sys.stderr)
```
Comment (fcurella) [inline vinnie/base.py]: What's the reason behind this change? I'd rather use `warnings` for more flexibility.

Comment (jsoref) [reply vinnie/base.py]: Expected user facing output shouldn't have stack traces nor should it have filenames & line numbers. This change suppresses the latter.

Comment (fcurella) [reply vinnie/base.py]: We can discuss this change separately. Could you file an issue for it?

Comment (jsoref) [reply vinnie/base.py]: Sure, split to #10 / #11

**Comment by fcurella** [comment]
Thank you! ✨
